WF-PCInstaller/python_installer/config/support_plugins.json

```diff
@@ -0,0 +1,158 @@
+{
+  "_comment": {
+      "standard_format": "see below",
+      "plugins": {
+          "APP_NAME": {
+              "app_search": "search_name",
+              "command": "commands that came with the app",
+              "conf_name": "plugin_conf_filename",
+              "AGENT": {
+                  "module": "python_module_name",
+                  "plugin_name": "For COLLECTD, this is name of plugin used. For TELEGRAF, this is the name to query sample config",
+                  "class_name": "installer_name"
+              }
+          }
+      }
+  },
+
+  "data": {
+      "plugins": {
+          "apache":
+          {
+              "app_search": "apache2|httpd",
+              "command": "apache2|httpd",
+              "conf_name": "wavefront_apache.conf",
+              "COLLECTD": {
+                  "module": "apache_plugin",
+                  "plugin_name": "apache",
+                  "class_name": "ApacheInstaller"
+              },
+              "TELEGRAF": {
+                  "module": "apache_configurator",
```

Comment:
I would just rename things so that `module` and `class_name` are same across telegraf and collect.d.  No need for things to diverge unnecessarily.